extern/cccl/CMakeLists.txt

```diff
@@ -44,3 +44,24 @@ set ( CUB_ENABLE_HEADER_TESTING OFF CACHE BOOL "Disable CUB header testing" )
 
 # Get it into the current directory.
 FetchContent_MakeAvailable( CCCL )
+
+# Check if <thrust/pair.h> works out of the box.
+include( CheckCXXSourceCompiles )
+set( CMAKE_REQUIRED_INCLUDES
+   "${CCCL_SOURCE_DIR}/thrust"
+   "${CCCL_SOURCE_DIR}/libcudacxx/include" )
+set( _THRUST_TEST_SOURCE "#include <thrust/pair.h>\nint main() { return 0; }" )
+check_cxx_source_compiles( "${_THRUST_TEST_SOURCE}" TRACCC_THRUST_WORKS )
+# If not, check if the _LIBCUDACXX_STRING_H_HAS_CONST_OVERLOADS flag makes it
+# work.
+if( NOT THRUST_PAIR_WORKS )
```

Comment:
Dang... Some issues were left behind from the copilot-recommended code... :frowning: